apm-server.docker.yml

```diff
@@ -279,6 +279,18 @@ apm-server:
     # Configure curve types for ECDHE based cipher suites.
     #ssl.curve_types: []
 
+  #---------------------------- APM Server - tail-based sampling ----------------------------
+
+  #sampling.tail:
+    #Set to `true` to enable tail based sampling. Disabled by default.
+    #enabled: false
+
+    #Synchronization interval for multiple APM Servers. Should be in the order of tens of seconds or low minutes.
+    #interval: 1m
+
+    #Criteria used to match a root transaction to a sample rate.
+    #policies: []
```

Comment:
Maybe we can add a commented policy? So users have an easier time writing these?

Reply:
I think the example is currently reflecting the default values. 

I think examples should go in the doc which already has an example configuration for TBS. WDYT ?

Reply:
Sure, let's leave it like that then